demo/admin/src/products/generator/generated/ProductForm.tsx

```diff
@@ -45,6 +44,7 @@ import { GQLManufacturersSelectQuery } from "./ProductForm.generated";
 import { GQLManufacturersSelectQueryVariables } from "./ProductForm.generated";
 import { CalendarToday as CalendarTodayIcon } from "@comet/admin-icons";
 import { FutureProductNotice } from "../../helpers/FutureProductNotice";
+import { Future_DateTimePickerField as DateTimePicker } from "@comet/admin";
```

Comment:
it should be `DateTimePickerField`, not `DateTimePicker`

Reply:
updated it